### DISCUSSION TITLE
codemirror-editor-vue3

### DISCUSSION THREAD

**Discussion by suijiaxin123** [rennzhang/codemirror-editor-vue3#23] [Q&A]
想问下大佬使用codemirror-editor-vue3怎么去实现调用find和替换的方法。谢谢了

**Comment by suijiaxin123** [comment]
![1656580837501](https://user-images.githubusercontent.com/54315100/176641623-207f12b6-62eb-4694-9188-c2697a58d48c.png)

**Reply by rennzhang** [reply]
https://codemirror.net/5/demo/search.html

**Reply by suijiaxin123** [reply]
好的 感谢大佬，后期codemirror-editor-vue3会对这一块进行方法封装吗

**Reply by rennzhang** [reply]
codemirror 本身的功能不会进行封装

**Reply by suijiaxin123** [reply]
好的  谢谢大佬
